### DISCUSSION TITLE
Missing backup/factory firmware

### DISCUSSION THREAD

**Discussion by SmartKobol** [tasmota/tasmotizer#105] [Q&A]
Dear Folks, I tasmotized (Tasmotizer! v.1.2 Windows version) a DIYBasicR3 (Sonoff). 9.4.0.3 was uploaded . I ticked the "Save original firmware". From the very beginning 3 bars appeared: Backup, Erase and Update. Backup procedure run and when finished (100%) stopped for a while and asked to disconnect device and press Ok. To carry on the flashing I plugged in the device (through FTDI) pressing down the only on/off small button on the device. The flashing procedure continued and successfully finished showing 3 filled status bar. After that I was searching the factory backup, without success. Where can I find anyhow? Windows file search did not find anything. (Earlier in case of a very first version of Sonoff Basic device the factory firmware saved in the same folder from where I uploaded the new tasmota version). If this R3 factory firmware gone, shall I use the early SO Basic one? I would like to save factory firmware of other SO (ZBBridge, 4CHPRO) when flashing, but I am a bit sceptic now. Can you suggest any safer solution or explanation?

**Comment by nlaja** [comment]
Backup creates a file according to the following pattern: backup_20231117_180552.bin
Mine was backed up here: C:\Users\%USERNAME%\AppData\Local\VirtualStore\Windows\SysWOW64\
